src/schema/README.md

```diff
@@ -0,0 +1,9 @@
+
+
+Why we models???
+
+[https://theholmesoffice.com/mongoose-connection-best-practice/](A good practice)
```

Comment:
be careful that guide is from 2013, Should we use a updated one?

Reply:
esquiusmi?

Reply:
O.O
https://www.youtube.com/watch?v=3D5tkAUNFa4